shoreline/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "90e1b0ed-0907-4973-929c-7e7f1be0c4f4",
+  "app_id": "shoreline-integration",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": true,
+  "tile": {
```

Comment:
Are these in place of "public_title", "categories", "type", "is_public", and "integration_id"?

Reply:
Yes, this is the new Manifest V2 format.